flask_pydantic/core.py

```diff
@@ -251,7 +251,7 @@ def wrapper(*args, **kwargs):
             res = func(*args, **kwargs)
 
             if response_many:
-                if is_iterable_of_models(res):
+                if is_iterable_of_models(res) or len(res) == 0:
```

Comment:
Can you please add a test case for this? 🙏

Reply:
i can't reproduce those error :smile: 
so much time has been gone

i'm afraid that was my mistake 
issue and pr can be closed